br/README.md

```diff
@@ -1,131 +1,156 @@
 
-# Test S3 compatible storage (MinIO).
-# Create a bucket to save backup by mc (a MinIO Client).
-mc config host add minio $S3_ENDPOINT $MINIO_ACCESS_KEY $MINIO_SECRET_KEY && \
-mc mb minio/mybucket
+### TiKV-BR Command Line Description
+A tikv-br command consists of sub-commands, options, and parameters.
 
-# Backup to S3 compatible storage.
-bin/br backup full --pd pd0:2379 --storage "s3://mybucket/full" \
-    --s3.endpoint="$S3_ENDPOINT"
+- Sub-command: the characters without - or --, including `backup`, `restore`, `raw` and `help`.
+- Option: the characters that start with - or --.
+- Parameter: the characters that immediately follow behind and are passed to the sub-command or the option.
+#### Backup Raw Data
+To back up the cluster raw data, use the `tikv-br backup raw` command. To get help on this command, execute `tikv-br backup raw -h` or `tikv-br backup raw --help`.
+For example, backup raw data in TiKV cluster to s3 `/backup-data/2022-09-16` directory.
 
-# Drop database and restore!
-mysql -uroot -htidb -P4000 -E -e "DROP DATABASE test; SHOW DATABASES;" && \
-bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
-    --s3.endpoint="$S3_ENDPOINT"
+```
+export AWS_ACCESS_KEY_ID=&{AWS_KEY_ID};
+export AWS_SECRET_ACCESS_KEY=&{AWS_KEY};
+tikv-br backup raw \
+    --pd "&{PDIP}:2379" \
+    -s "s3://backup-data/2022-09-16/" \
+    --ratelimit 128 \
+    --dst-api-version v2 \
+    --log-file="/tmp/br_backup.log
+```
+Explanations for some options in the above command are as follows:: 
+- `backup`: Sub-command of `tikv-br`.
+- `raw`: Sub-command of `backup`.
+- `-s` or `--storage`: Storage of backup files.
+- `"s3://backup-data/2022-09-16/"`: Parameter of `-s`, save the backup files in `"s3://backup-data/2022-09-16/"`.
+- `--ratelimit`: Specifies the maximum speed at which a backup operation is performed on each `TiKV` node.
+- `128`: The value of `ratelimit`, unit is MiB/s.
+- `--pd`: Service address of `PD`.
+- `"${PDIP}:2379"`:  Parameter of `--pd`.
+- `--dst-api-version`: The `api-version`, please see [tikv-server config](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#api-version-new-in-v610) of backup files.
+- `v2`: Parameter of `--dst-api-version`, the optionals are `v1`, `v1ttl`, `v2`(Case insensitive). If no `dst-api-version` is specified, the `api-version` is the same with TiKV cluster of `--pd`.
+
+A progress bar is displayed in the terminal during the backup. When the progress bar advances to 100%, the backup is complete. The progress bar is displayed as follows:
+```
+br backup raw \
+    --pd "${PDIP}:2379" \
+    --storage "s3://backup-data/2022-09-16/" \
+    --log-file backupfull.log
+Backup Raw <---------/................................................> 17.12%.
 ```
```

Comment:
Suggest to also introduce the result message of backup later.
(Especially the usage of `BackupTs`. And the kv size was confused to users in some past cases.)

Reply:
Added, PTAL~, thanks a lot.